webapp/channels/src/components/edit_channel_header_modal/edit_channel_header_modal.tsx

```diff
@@ -227,12 +227,19 @@ export class EditChannelHeaderModal extends React.PureComponent<Props, State> {
                 </Modal.Header>
                 <Modal.Body bsClass='modal-body edit-modal-body'>
                     <div>
-                        <p>
+                        <label
+                            htmlFor='edit_textbox'
+                            id='edit_textbox_label'
+                            style={{
```

Comment:
Could you move this out of a style tag and into the CSS for this component? I think it could go under `.edit-modal-body` in `sass/components/_modal.scss`